src/dsp_tools/resources/schema/project.json

```diff
@@ -769,30 +765,46 @@
                     },
                     "then": {
                         "properties": {
+                            "super": {
+                                "type": "array",
+                                "items": {
+                                    "type": "string",
+                                    "oneOf": [
+                                        {
+                                            "enum": [
+                                                "hasValue"
+                                            ]
+                                        },
+                                        {
+                                            "$ref": "#/definitions/prefixedname_or_uri"
+                                        }
+                                    ]
+                                }
+                            },
                             "gui_element": {
-                                "super": {
```

Comment:
This here was quite certainly a bug. `super` should not be subordinate of `gui_element`.